internal/metrics/logfilemetricexporter/metric_exporter.go

```diff
@@ -0,0 +1,50 @@
+package logfilemetricexporter
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/cluster-logging-operator/internal/metrics/telemetry"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+
+	log "github.com/ViaQ/logerr/v2/log/static"
+	loggingv1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	loggingv1alpha1 "github.com/openshift/cluster-logging-operator/apis/logging/v1alpha1"
+	"k8s.io/client-go/tools/record"
+	client "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+)
+
+func Reconcile(lmfeInstance *loggingv1alpha1.LogFileMetricExporter,
+	requestClient client.Client,
+	er record.EventRecorder,
+	clInstance *loggingv1.ClusterLogging) error {
+
+	if err := ReconcileService(er, requestClient, lmfeInstance.Namespace, constants.LogfilesmetricexporterName, utils.AsOwner(clInstance)); err != nil {
```

Comment:
I think owner here needs to be the API object.  The main case is you create a LFME object.  The upgrade case is you create CL or CLF.  It will have to be whichever initiated the reconcile

Reply:
Changed so that ownership is dependent on which object initiated the reconcile. It will either be the defined `LFME` or the `ClusterLogging` object for the "virtual" LFME.